### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by WillBeebe**

### CODE REVIEW

File: pkg/bundle/templates/terraform/README.md [hunk 1]
```diff
@@ -2,53 +2,18 @@
...
+  
+## Developing  
+  
+### How To Develop A Bundle
+  
+To learn how to develop a bundle for Massdriver, check out our [bundle docs](https://docs.massdriver.cloud/bundles/development).
+  
+### Contribution guidelines
+
+So you're interested in contributing to Massdriver Bundles?  Please refer to Massdruver's overall
```
Comment (mclacore) [inline pkg/bundle/templates/terraform/README.md:17]:
```suggestion
So you're interested in contributing to Massdriver Bundles?  Please refer to Massdriver's overall
```

**Review by mclacore — CHANGES_REQUESTED** [review]

**Comment by coryodaniel** [comment]
@chrisghill is importing the new template into the CLI so customers get it to. It'll include `PARAMS:START` fences as well so that @jaketf's python script can keep users params, connections, and artifact docs up to date in the readme.

New template is pretty tight -> https://github.com/massdriver-cloud/aws-vpc

**Comment by jaketf** [comment]
@coryodaniel I'm a bit confused. On Friday I thought this was supposed to be the "go forward" bundle repo template. that aws-vpc readme does look dope is there a PR for that template @chrisghill is using somewhere? I can work on getting the python script into our workflow but I wanna make sure I'm starting in the right place :)

**Comment by coryodaniel** [comment]
He’s been working on putting the new template together. We rolled it to all the pre-existing bundles. They have the PARAMS:START tokens in them. He’s replacing the README here w the same template. 

We’ll put the Python script in precommit (existing and this template for in the cli) so it keeps those PARAMS and CONNECTIONS sections updated

**Comment by jaketf** [comment]
@WillBeebe seems we should we close this PR then.

What was the point of the scramble Friday [tracked in this sheet](https://docs.google.com/spreadsheets/d/1U4IDB-rwA2s-gVs-hCPTknYq3Y3_Nvfa4GMo8PykWsE/edit#gid=2076264696) to update READMEs with this template? Seems we wasted effort on opening over a dozen PRs that now near useless because they were using the wrong readme template.

Some of those PRs also updated massdriver.yamls / pre-commits though so now we should till through them and make them only scoped to those changes.f

cc @mclacore

**Comment by WillBeebe** [comment]
Yeah this PR is no good anymore. I opened it as tracking for the readme edits. I thought this would be reviewed, approved, merged, then we'd copy over to the bundle repos. 

Every bundle PR I opened for public launch has Readme edits so I'll toss those and just keep the massdriver yaml edits

**Pull request closed without merging**